<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.yuYEQP7' shared service.

return $this->privates['.service_locator.yuYEQP7'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('atendimento' => function (): \Novosga\Entity\Atendimento {
    return ($this->privates['.errored..service_locator.yuYEQP7.Novosga\Entity\Atendimento'] ?? $this->load('getAtendimento5Service.php'));
}, 'atendimentoService' => function (): \Novosga\Service\AtendimentoService {
    return ($this->services['Novosga\Service\AtendimentoService'] ?? $this->load('getAtendimentoServiceService.php'));
}, 'translator' => function () {
    return ($this->services['translator'] ?? $this->getTranslatorService());
}));
